api.js

```diff
@@ -25,3 +25,29 @@ export const fetchAllComments = (article_id) => {
     return data;
   });
 };
+
+export const upVote = (article_id) => {
+  return ncNewsApi
+    .patch(`/articles/${article_id}`, {
+      inc_votes: 1,
+    })
+    .then((response) => {
+      console.log(response);
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
```

Comment:
This is not where we should be trying to deal with any of the .catch(...) errors. If you do try to do this here you are allowing this promise to resolve and enter the .then(...) in your component file which is not ideal.